core/src/main/java/org/apache/iceberg/rest/auth/OAuth2Util.java

```diff
@@ -381,5 +394,42 @@ public Pair<Integer, TimeUnit> refresh(RESTClient client) {
 
       return null;
     }
+
+    @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
What is the issue this is suppressing?

Reply:
I see it was there before. I don't remember seeing this though. Anyway, it's fine.